clj/src/cljd/build.clj

```diff
@@ -337,11 +343,15 @@
 
 (defn -main [& args]
   (let [f (java.io.File. (System/getProperty "user.dir") "cljd.edn")
+        is-windows? (str/includes? (str/lower-case (or (System/getProperty "os.name") "")) "windows")
         config (if (.exists f)
                  (with-open [rdr (-> f io/reader java.io.PushbackReader.)]
                    (edn/read rdr))
                  {})]
     (binding [*ansi* (and (System/console) (get (System/getenv) "TERM"))
+              *may-dart* (if is-windows? ["dart.exe" "dart.bat"] *may-dart*)
```

Comment:
Why do we have to test both exe and bat? It depends from Dart version to another?

Reply:
My understanding is that dart installation provides dart.exe, but flutter installation provides dart.bat

Reply:
Yes, flutter sdk provides dart sdk, but it does not directly expose dart.exe, but provides dart.bat to link to dart.exe, dart.bat is located in the flutter bin directory, so you only need to add this directory to Path You can execute flutter and dart on the command line. Another case is pure Dart users, then they have a dart.exe directly instead of dart.bat.

Reply:
I committed a more localized patch, could you confirm it works?